code/parachain/frame/liquid-staking/relayer/src/main.rs

```diff
@@ -1,46 +1,96 @@
 use std::{io::Bytes, str::FromStr, sync::Arc, time::Duration};
 use subxt::{
-	config::Header, dynamic::Value, ext::scale_value::Composite, rpc::types::StorageChangeSet,
-	tx::Payload, Config, OnlineClient, SubstrateConfig,
+	config::Header,
+	dynamic::Value,
+	ext::scale_value::Composite,
+	rpc::types::StorageChangeSet,
+	tx::{PairSigner, Payload},
+	utils::AccountId32,
+	Config, OnlineClient, SubstrateConfig,
 };
-// use subxt_signer::sr25519::dev::{self};
-use crate::parachain_subxt::api::runtime_types::pallet_liquid_staking::types::{
-	StakingLedger, UnlockChunk,
-};
-use futures_util::stream::StreamExt;
-use sp_core::Pair;
-use sp_keyring::AccountKeyring;
-use subxt::{tx::PairSigner, utils::AccountId32};
+
+struct SovereignSubAccount {
+	pub address: String,
+	pub storage_key: StorageKey,
+	pub derivative_index: u16,
+}
 
 #[tokio::main]
 async fn main() {
-	let sovereign_account_id = "13YMK2ecbyxtm4cmFs31PqzWmQ7gWVboJSmXbcA56DB94xB9";
-	let sovereign_account_id_index_0 = "12x6QU4c9eRPxJMATFsRNFiZTMK5QgZkdZFFeu2QDKn4TR82";
-	let sovereign_account_id_index_1 = "1461Z7Bm1bwQpz1PuYMQ8phj9bRpxNU7ZYsb7aXQRAUuNecG";
-	let sovereign_account_id_index_2 = "15ySsNFkAhswdn9hSKkzoK7LhmJrj8bgyUZQAiM7Df9JpBUH";
-	let sovereign_account_id_index_3 = "15s3DuzMeftBH7YdHykwPDUd2DBxdNbiyqgDfZDA3i5eRwUW";
-	let sovereign_account_id_index_4 = "12uNvUSK39SDbHbqWuMhFdw2hHySrkbenVrHdS678fkj9BBb";
-	let sovereign_account_id_index_5 = "14tDkT3U93Pc1wLrHEjfYuhPPnFpMwDr7o8phPCTwTRj5wfE";
-
-	let para_storage_key = parachain_header_storage_key(2019);
-
-	let p0 = "5f3e4907f716ac89b6347d15ececedca422adb579f1dbf4f3886c5cfa3bb8cc456d08aea5b028f73699523ae21709a815640ec97748f5b5da9a2298e830e8971df7908861e1710b957fe06f0703bca7d";
-	let p1 = "5f3e4907f716ac89b6347d15ececedca422adb579f1dbf4f3886c5cfa3bb8cc491af1d8906a21795a98d84506b4216828886ca7474c66c027a9dc5d73901481568d551d01f13d0eb3bd36dd20ed2f13e";
-	let p2 = "5f3e4907f716ac89b6347d15ececedca422adb579f1dbf4f3886c5cfa3bb8cc4bc407118eef848de28bad224cc60c8fedbfdc1d87d5f65f94602d8e31d7426de7b7d0bb4a8e9223b72bb39231737377c";
-	let p3 = "5f3e4907f716ac89b6347d15ececedca422adb579f1dbf4f3886c5cfa3bb8cc43802b39b0fa95ce7e37486987631cef3d71aafa64aa8d5d9164071a14106cdba71fe4b8fc08de39a7606c8d16f5b20f8";
-	let p4 = "5f3e4907f716ac89b6347d15ececedca422adb579f1dbf4f3886c5cfa3bb8cc4b8ac2e6f27924e06455d8e5c75b6a775542eca539ca4e92c4b4e0496bef934595597645e21eb29b5ce94bc38b7e45181";
-	let p5 = "5f3e4907f716ac89b6347d15ececedca422adb579f1dbf4f3886c5cfa3bb8cc4232647f67d6923193db6aedafe0d5ea3abc522231610ec74f6391785db08c68f3c07c1bb45e2193b413f66926dec5072";
+	//RELAY_HOST=wss://kusama-rpc.polkadot.io:443 PARA_HOST=ws://127.0.0.1:8000 cargo run -p
+	// lsd-relayer
+
+	cfg_if::cfg_if! {
+		if #[cfg(feature = "composable")] {
+			println!("{}", "Composable");
+			let sovereign_account_id = "13YMK2ecbyxtm4cmFs31PqzWmQ7gWVboJSmXbcA56DB94xB9";
+			let sovereign_account_id_index_0 = "12x6QU4c9eRPxJMATFsRNFiZTMK5QgZkdZFFeu2QDKn4TR82";
```

Comment:
why are all of these accounts hardcoded? what are these values

Reply:
this is sovereign sub accounts for each of the network. for picasso and for composable.

Reply:
i remove changes of relayer. 
relayer changes in other pr.